### PR TITLE
FIx `exchange_rate` when empty string

### PR DESCRIPTION
**Pull request by vnovitskyi**
Fix exchange_rate when an empty string

### CODE REVIEW

File: lib/cfonb/operation_detail/mmo.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def self.apply(operation, line)
 
         operation.original_amount = sign * BigDecimal(line.detail[4..17]) / (10**scale)
         exchange_rate_value = line.detail[26..29]
-        return unless exchange_rate_value
+        return if exchange_rate_value.nil? || exchange_rate_value.empty?
```
Comment (frantisekrokusek) [inline lib/cfonb/operation_detail/mmo.rb]: what about this ?
`
exchange_rate_value = line.detail[26..29].to_i
return if exchange_rate_value.zero?
`

**Review by frantisekrokusek — CHANGES_REQUESTED** [review]
suggestion
please add specs

File: lib/cfonb/operation_detail/mmo.rb (hunk 1 above)
Comment (vnovitskyi) [reply lib/cfonb/operation_detail/mmo.rb]: to_i and to_f is not working here.
I added specs. pls review.
do you know what `04` means here `EUR200000001875625             04` ?

Comment (frantisekrokusek) [reply lib/cfonb/operation_detail/mmo.rb]: are you sure this is on the same line ? looks like a start of a line

Comment (vnovitskyi) [reply lib/cfonb/operation_detail/mmo.rb]: I downloaded all statements and parsed them. This is what we have an operation line. I can send them to you.

**Review by frantisekrokusek — APPROVED** [review]

**Pull request merged**